pkg/image/api/helper.go

```diff
@@ -847,3 +848,41 @@ func SetContainerImageEntrypointAnnotation(annotations map[string]string, contai
 func LabelForStream(stream *ImageStream) string {
 	return fmt.Sprintf("%s/%s", stream.Namespace, stream.Name)
 }
+
+// ImageHasSignature returns true if the signature identified by type and blob is already stored in the image.
+func ImageHasSignature(image *Image, sType string, blob []byte) bool {
```

Comment:
Moved.
On 06/06/2016 05:20 PM, David Eads wrote:
>
> In pkg/image/api/helper.go <https://github.com/openshift/origin/pull/8371#discussion_r65909746>:
>
> > @@ -847,3 +848,41 @@ func SetContainerImageEntrypointAnnotation(annotations map[string]string, contai
> >  func LabelForStream(stream *ImageStream) string {
> >  	return fmt.Sprintf("%s/%s", stream.Namespace, stream.Name)
> >  }
> > +
> > +// ImageHasSignature returns true if the signature identified by type and blob is already stored in the image.
> > +func ImageHasSignature(image *Image, sType string, blob []byte) bool {
>
> Move these to the next pull? I have questions about them, like: "why don't I use a signature type", that probably 
> don't need to be addressed here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/openshift/origin/pull/8371/files/4371c2172d2421e632faa67cf851acce66753c04#r65909746>, or mute the 
> thread <https://github.com/notifications/unsubscribe/ABaQk5295ZdAXs30j5CfIputhZi2FiXoks5qJDq5gaJpZM4IBAFQ>.
>